test/utils/test-apis.js

```diff
@@ -8,14 +9,16 @@ const IPFS = require('ipfs')
 // https://github.com/ipfs/js-ipfsd-ctl#ipfsfactory---const-f--ipfsfactorycreateoptions
 let jsIpfs = {
   'js-ipfs': {
-    type: 'proc', 
+    type: 'proc',
     exec: IPFS,
+    IpfsClient: IPFSHTTPClient,
```

Comment:
Thanks for updating all of this!

Question here - probably something I don't fully understand but it seems counterintuitive to use the HTTP Client for `js-ipfs`? Is there something significant that changed under the hood?

Reply:
I believe this to be a misunderstanding. See https://github.com/ipfs/js-ipfs/issues/2446#issuecomment-536202616 and https://github.com/ipfs/js-ipfsd-ctl/pull/384

Reply:
The only way it would make sense is if we added support to spawn the node inside a Service/Web Worker -- https://github.com/ipfs/js-ipfsd-ctl/issues/385 -- and that would have to use https://github.com/ipfs-shipyard/ipfs-postmsg-proxy

Reply:
Thanks David. Am I right in thinking that removing that line would make no difference?

Reply:
You are right indeed :)

Reply:
Quite right, this line is not necessary.

It's a good idea to specify it to the `go` daemon further down though, if you are expecting the value returned from the daemon factory to have the same API as the module passed as the `exec` arg in the `proc` version, because otherwise you will get the `ipfs-http-client` bundled with the version of `ipfsd-ctl` you are using, which may not have exactly the same API as the js `ipfs` implementation.

For example `ifpsd-ctl@0.42.3` depends on `ipfs-http-client@32.0.1` but `ipfs@0.38.0-rc.6` depends on (and is only tested with) `ipfs-http-client@37.0.2` so you should keep these in sync if you want to seamlessly switch between js/go implementations as you do here.